lxdock/conf/config.py

```diff
@@ -115,6 +153,27 @@ def _get_container_config_dict(self, container_dict):
         del container_config['containers']
         return container_config
 
+    def _get_dotenv_items(self, env_filepath):
```

Comment:
Is the `.env` format supposed to be a standard of some sort? According to rules at https://github.com/bkeepers/dotenv, we're missing a few cases. People using `.env` file that aren't specific to lxdock will probably expect those to be parsed as if the file was interpreted in `bash` and this function will never stop growing more and more complex.

Couldn't we simply `source` the file?